src/AppBundle/Resources/views/themes/app/page_path.html.twig

```diff
@@ -1,13 +1,17 @@
 {% if show_path and path_array|length > 1 %}
-    <div class="breadcrumb-wrapper">
-        <div class="container">
-            <ol class="breadcrumb">
-                {% for path_item in path_array %}
-                    {% if path_item.url != false %}
-                        <li><a href="{{ path_item.url }}">{{ path_item.text }}</a></li>
-                    {% endif %}
-                {% endfor %}
-            </ol>
-        </div>
-    </div>
+    <nav class="breadcrumb-wrapper" aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            {% for path_item in path_array %}
+                {% if path_item.url != false and path_item.location.contentInfo.contentTypeIdentifier not in ezpublish.configResolver.parameter('container_content_types', 'ngsite') %}
+                    <li class="breadcrumb-item">
+                        <a href="{{ path_item.url }}">{{ path_item.text }}</a>
+                    </li>
+                {% else %}
+                    <li class="breadcrumb-item active" aria-current="page">
```

Comment:
Actually, if what @Ljudevit says is true, than class `active` is not correct here, because it would be applied to ng_container and folder too.

Reply:
I believe this "else" condition is made to create an item in the end on the last place, which is basically {{location}}.

I believe that the loop should:

1. iterate through location path
2. ignore all container/folder objects
3. ignore current location

and in the end, output the {{ location.name }}

That way we would avoid any issues with conditions.

Reply:
Not really, the logic before was okay. The backend generates items and assigns the URLs as needed. It actually uses `path_helper.excluded_content_types` option. So what this template needs to do is leave the logic in the loop as before.

Reply:
So it means that:
`path_item.location.contentInfo.contentTypeIdentifier not in ezpublish.configResolver.parameter('container_content_types', 'ngsite')`

is not necessary and can be removed from the condition, and with that in mind, everything else will work just peachy, right?

Reply:
If the logic not to display `ng_container` and `folder` **at all** (as opposed to displaying them without URL) in the breadcrumb is what you need, then yes.